sift/client.py

```diff
@@ -313,6 +314,135 @@ def get_workflow_status(self, run_id, timeout=None):
         except requests.exceptions.RequestException as e:
             raise ApiException(str(e))
 
+    def get_decisions(self, entity_type, limit=None, start_from=None, abuse_types=None, timeout=None):
+        """Get decisions available to customer
+
+        Args:
+            entity_type: only return decisions applicable to entity type {USER|ORDER}
+            limit: number of query results (decisions) to return [default: 100]
+            start_from: result set offset for use in pagination [default: 0]
+            abuse_types: csv of abuse_types to filter returned decisions by (optional)
```

Comment:
The default behavior is to return decisions of all abuse_types. Is it clearer to add a note describing default behavior to the effect `default: all abuse types returned` or `default: no filter applied` ?